README.md

```diff
@@ -1,8 +1,60 @@
-# MFSDA_Python
+# MFSDA: Multivariate Functional Shape Data Analysis
 
-This MFSDA_Python package is developed by Chao Huang and Hongtu Zhu from the [BIG-S2 lab](http://odin.mdacc.tmc.edu/bigs2/). 
+MFSDA_Python is a Python based package for statistical shape analysis.
 
-MFSDA_Python is a Python based package for statistical shape analysis. A multivariate varying coefficient model is introduced to build the association between the multivariate shape measurements and demographic information and other clinical variables. Statistical inference, i.e., hypothesis testing, is also included in this package, which can be used in investigating whether some covariates of interest are significantly associated with the shape information. The hypothesis testing results are further used in clustering based analysis, i.e., significant suregion detection. 
+A multivariate varying coefficient model is introduced to build the association between the multivariate shape measurements and demographic information and other clinical variables.
+
+Statistical inference, i.e., hypothesis testing, is also included in this package, which can be used in investigating whether some covariates of interest are significantly associated with the shape information.
+
+The hypothesis testing results are further used in clustering based analysis, i.e., significant suregion detection.
+
+## Modules
+
+### Loadable module
+
+The project a Slicer loadable allowing to XXX that internally leverage invokes command line modules described below.
+
+
+### CLI modules
+
+| CLI Name           | Description |
+|--------------------|-------------|
+| [MFSDA_createShapes][MFSDA_createShapes]             | ??? |
+| [MFSDA_run][MFSDA_run]                               |  ??? |
+| [MFSDA_selectVariablesPCA][MFSDA_selectVariablesPCA] | Variable selection using Correlation with Principal Components. |
+
+[MFSDA_createShapes]: https://github.com/DCBIA-OrthoLab/MFSDA_Python/blob/master/MFSDA/MFSDA_createShapes.xml
+[MFSDA_run]: https://github.com/DCBIA-OrthoLab/MFSDA_Python/blob/master/MFSDA/MFSDA_run.xml
+[MFSDA_selectVariablesPCA]: https://github.com/DCBIA-OrthoLab/MFSDA_Python/blob/master/MFSDA/Resources/Libraries/MFSDA_selectVariablesPCA.py
+
+
+## History
+
+This project was initially developed in early August 2017 by Chao Huang and Hongtu Zhu from
+the [BIG-S2 lab](https://www.med.unc.edu/bigs2/) at the University of North Carolina at Chapel Hill School of Medicine.
```

Comment:
I changed the URL from http://odin.mdacc.tmc.edu/bigs2/ to https://www.med.unc.edu/bigs2/